src/odoc/indexing.ml

```diff
@@ -23,21 +23,29 @@ let parse_input_files input =
     (Ok []) input
   >>= fun files -> Ok (List.concat files)
 
-let compile_to_json ~output ~occurrences hierarchies =
+let compile_to_json ~output ~occurrences ~wrap ~simplified hierarchies =
   let output_channel =
     Fs.Directory.mkdir_p (Fs.File.dirname output);
     open_out_bin (Fs.File.to_string output)
   in
   let output = Format.formatter_of_out_channel output_channel in
+  if wrap then Format.fprintf output "let documents = ";
+  Format.fprintf output "[";
   let all =
     List.fold_left
       (fun acc hierarchy ->
         Tree.fold_left
-          ~f:(fun acc entry -> Json_search.of_entry ?occurrences entry :: acc)
+          ~f:(fun acc entry ->
+            Json_search.of_entry ~simplified ?occurrences entry :: acc)
           acc hierarchy)
       [] hierarchies
   in
   Format.fprintf output "%s" (Odoc_utils.Json.to_string (`Array (List.rev all)));
+  if wrap then
+    Format.fprintf output
+      ";\n\
+       const options = { keys: ['name', 'comment'] };\n\
+       var idx_fuse = new Fuse(documents, options);\n";
```

Comment:
If we are making that the use of Fuse part of odoc (and not part of `odoc_driver`) than I think we should add it to `html_support_files`. Personally, I would prefer to have Fuse use part of `odoc_driver` (although I'm fine with both solutions).

Reply:
I think this is a temporary grotty hack that I'd like to go away as soon as we've got proper sherlodoc support in ocaml.org. I'll see about hiding the options in the CLI.

Reply:
If it is something we are not going to keep, why not making it generated by the driver, from the output json file? This way, no need to hide it, and we have a guarantee no-one relies on it, if it happens to stay a bit longer before being removed.

Reply:
(This could be done after the beta release, but before the actual release!